rice-datanode/CMakeLists.txt

```diff
@@ -1,10 +1,13 @@
-include_directories(${CMAKE_SOURCE_DIR}/rpcserver ${CMAKE_SOURCE_DIR}/utility/config-reader ${CMAKE_SOURCE_DIR}/utility/logging)
+include_directories(${CMAKE_SOURCE_DIR}/rpcserver ${CMAKE_SOURCE_DIR}/utility/pugixml/src ${CMAKE_SOURCE_DIR}/utility/logging ${CMAKE_SOURCE_DIR}/zookeeper/include ${CMAKE_SOURCE_DIR}/zkwrapper ${CMAKE_SOURCE_DIR}/native-filesystem )  
```

Comment:
It looks like all of the changes from commit c744124 to this file, ClientDatanodeProtocolImpl.cc, ClientDatanodeProtocol.h, and rice-datanode/main.cc were reverted when zookkerrplus was merged in. These changes should not have been reverted, as we want to be using the config reader and easylogging++